airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -476,7 +476,9 @@ def run_check_dbt_command(self, normalization_image: str, command: str, cwd: str
         """
         Run dbt subprocess while checking and counting for "ERROR", "FAIL" or "WARNING" printed in its outputs
         """
-        if normalization_image.startswith("airbyte/normalization-oracle") or normalization_image.startswith("airbyte/normalization-clickhouse"):
+        if normalization_image.startswith("airbyte/normalization-oracle") or normalization_image.startswith(
+            "airbyte/normalization-clickhouse"
+        ):
```

Comment:
small not totally necessary optimization for readability
```suggestion
             if any([normalization_image.startswith(x) for x in ["airbyte/normalization-oracle", "airbyte/normalization-clickhouse"]]):
```

Reply:
done